docs/content/python/getting-started/application/_index.md

```diff
@@ -0,0 +1,40 @@
+---
+title: Add Oso to Your Python App
+weight: 2
+description: |
+    In this guide, we'll cover the basics of adding Oso to a Python application
+    to enforce authorization. Start here to understand the basics of using
+    Oso. Then, jump into our more detailed how to section on topics that are
+    important to you.
+aliases:
+  - /getting-started/application/index.html
+---
+
+# Add Oso to Your Python App
+
+In this guide, we'll cover the basics of adding Oso to your Python application
```

Comment:
I don't it's necessary to tell people up front this is "the basics". 

E.g. Here: We'll cover how to add Oso to your Python application. Later we can link to deeper guides.
(Same comment for the page description).